test/promql/tests/data_plane/pipeline_to_snapshot_test.yaml

```diff
@@ -33,6 +33,7 @@ tests:
               Time from pipeline run finished to snapshot marked in progress has been over
               30s for more than 10% of requests during the last 5 minutes on cluster
               cluster01
+            alert_routing_key: pipelines
```

Comment:
this alert is owned by the integration service team per https://redhat-internal.slack.com/archives/C056R76M1EJ/p1706541712730629 and https://redhat-internal.slack.com/archives/C04FDFTF8EB/p1706543295126809

with the mapping that @amisstea just provided me in gitlab, it looks like integration-service is the correct tag and not the plnsvc-tests that was employed in the past, or the piipelines tage noted here